targets/targets.json

```diff
@@ -6558,17 +6543,9 @@
                 "IAR"
             ]
         },
-        "device_has_add": [
-            "MPU",
-            "FLASH"
-        ],
-        "device_has_remove": [
-            "SERIAL_FC"
-        ],
-        "release_versions": [
-            "2",
-            "5"
-        ],
+        "device_has_add": ["MPU", "FLASH"],
```

Comment:
Can you rebase and fix the styling? It was updated on master

Reply:
Is there any way to fix the styling automatically, like `astyle` ?

Reply:
We run astyle but it doesn't auto fix so you will need to do it manually for now. Though nothing stopping you running astyle locally and have it do it for you....